src/zsh_completion/_firejail.in

```diff
@@ -160,8 +160,6 @@ _firejail_args=(
     '*--seccomp.32.keep=-[enable seccomp filter, and whitelist the 32 bit syscalls specified by the command]: :'
     # FIXME: Add errnos
     '--seccomp-error-action=-[change error code, kill process or log the attempt]: :(kill log)'
-    '--shell=none[run the program directly without a user shell]'
-    '--shell=-[set default user shell]: :_values $(cat /etc/shells)'
```

Comment:
Was shell removed too?

Reply:
> Was shell removed too?

Yes:

```console
$ firejail --quiet --shell=/bin/sh true; echo $?
Warning: --shell feature has been deprecated
1
$ cat test.profile
shell none
shell /bin/sh
$ firejail --quiet --profile=./test.profile true
Warning: "shell none" command in the profile file is done by default; the command will be deprecated
Error: line 2 in ./test.profile is invalid
```

(The first warning is actually an error)